phpstan.neon

```diff
@@ -1,3 +1,5 @@
 parameters:
-  level: 0
+  level: 1
   bootstrap: %currentWorkingDirectory%/vendor/autoload.php
+  universalObjectCratesClasses:
```

Comment:
What is this setting about?

Reply:
It declares to phpstan that any object of those classes is a universal crate, on which you can get and set any property even though it doesn't have those properties natively (through `__get` and `__set`). 'stdClass' is the most common example, but `Component` is one too.

If you wouldn't have this, you'd have to add an `@property` annotation for every possible value, like VEVENT, DTSTART, SUMMARY, etc.

Reply:
Ahh I tried in the past defining all properties :-)

Reply:
I tried that too on my first attempt ;-)